src/hotspot/share/jvmci/jvmciCompilerToVMInit.cpp

```diff
@@ -26,10 +26,14 @@
 #include "compiler/compiler_globals.hpp"
 #include "compiler/oopMap.hpp"
 #include "gc/shared/barrierSet.hpp"
+#include "gc/shared/barrierSetAssembler.hpp"
 #include "gc/shared/cardTable.hpp"
 #include "gc/shared/collectedHeap.hpp"
 #include "gc/shared/gc_globals.hpp"
 #include "gc/shared/tlab_globals.hpp"
+#include "gc/shared/barrierSetNMethod.hpp"
+#include "gc/z/zThreadLocalData.hpp"
+#include "gc/z/zBarrierSetRuntime.hpp"
```

Comment:
Should this be guarded with an INCLUDE_ZGC check? This also needs to be sorted.

Reply:
Guarded the includes and the code which uses them.